overlord/snapstate/snapmgr_test.go

```diff
@@ -641,7 +642,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 		},
 		{
 			op:   "copy-data",
-			name: "/snap/some-snap/11",
+			name: fmt.Sprintf("%s/some-snap/11", dirs.SnapMountDir),
```

Comment:
Is there a real benefit in doing this?  Why isn't that path hardcoded for these tests?